argilla/docs/how_to_guides/workspace.md

```diff
@@ -84,6 +84,17 @@ client = rg.Argilla(api_url="<api_url>", api_key="<api_key>")
 retrieved_workspace = client.workspaces("my_workspace")
 ```
 
+You can also use the user `id` to fetch the workspace:
+```python
+import argilla as rg
+
+client = rg.Argilla(api_url="<api_url>", api_key="<api_key>")
+
+workspace = client.workspaces("<uuid-or-uuid-string>")
```

Comment:
does this work? it seems you are using two arguments `user` and `uuid` also shouldn't `uuid-or-uuid-string` be `uuid-or-uuid-string-or-name`?